### PR TITLE
fix: Scroll into view doesn't work any more for scrollable divs 

### LINKED ISSUE #4631: Scroll into view doesn't work any more for scrollable divs
Discord thread https://discord.com/channels/955905230107738152/1319754382908199025

For some reason .scrollIntoView only works now for document-level scrolling, but doesn't work for containers.

For simple repro:

1. add a flex container, set overflow to hidden and width to 100vw
2. add a bunch of elements to it so that they take more space than you can see
3. select in navigator one of the elements and see canvas won't scroll to the elements

### PR DESCRIPTION
**Pull request by istarkov**
## Description

closes  #4631

tested in Safari, FF, Chrome

## TODO:

- [x] - Menu content cause
image

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 0000)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: apps/builder/app/shared/dom-utils.ts [hunk 1]
```diff
@@ -216,3 +217,132 @@ export const getAllElementsBoundingBox = (
...
+    visibility: "hidden",
+  });
+  container.appendChild(testDiv);
+  testDiv.setAttribute(doNotTrackMutationAttribute, "true");
+
+  const { left, top, width, height } = testDiv.getBoundingClientRect();
+  container.removeChild(testDiv);
+
+  const x1 = left,
+    y1 = top;
```
Comment (TrySound) [inline apps/builder/app/shared/dom-utils.ts:266]: maybe write explicit const here?

**Review by TrySound — APPROVED** [review]

File: apps/builder/app/shared/dom-utils.ts (hunk 1 above)
Comment (istarkov) [reply apps/builder/app/shared/dom-utils.ts]: done

**Review by kof — APPROVED** [review]

**Comment by kof** [comment]
Added swiper template and clicking on those content areas still doesn't scroll https://p-9b2603c4-d3cb-49d6-9fe2-85a3f1678891-dot-radix-collapse.development.webstudio.is/

**Review by kof — CHANGES_REQUESTED** [review]
Added swiper template and clicking on those content areas still doesn't scroll https://p-9b2603c4-d3cb-49d6-9fe2-85a3f1678891-dot-fix-scroll.development.webstudio.is/?pageId=DgdAm2lPUAwdgekIHZrpI

**Review by kof — APPROVED** [review]

**Pull request merged**